src/Material.cpp

```diff
@@ -1313,7 +1452,7 @@ void Material::checkSigmaT() {
  *          scattering cross-sections and chi for all energy groups.
  * @return character array of this Material's attributes
  */
-std::string Material::toString() {
+std::string MacroMaterial::toString() {
```

Comment:
I think this should be pretty easy to implement. This will fall under the rug after this PR, so I think it's best to get the implementation for it right (as generalized as possible) with this PR.